Automattic/woocommerce-payments
# Deposits information block on account overview page

Fixes #1410 

#### Changes proposed in this Pull Request

Introduce a new block on Account Overview page: deposits information. It contains data similar to one found on the Deposits page but provides a separate block for each deposit currency.

It also contains some changes to the TypeScript tooling set-up to improve compatibility between linters provided by different tools.

Task list:

 - [x] Create a mark-up for a deposit information block
 - [x] Create a block per deposit currency once deposit data is loaded
 - [x] Extract supporting functions into a separate module in `utils`
 - [x] Add the instant deposit link
 - [x] Update/add styles to match the Figma files.
 - [x] Add more TypeScript declarations. This could easily be a separate issue. (https://github.com/Automattic/woocommerce-payments/issues/1967)







#### Testing instructions

* Checkout out server PR 837-gh-Automattic/woocommerce-payments-server if it has not already been merged.
 * Enable option _wcpay_feature_account_overview
 * Open wp-admin and go under **Payments -> Overview**
 * Ensure that there is at least one deposit information block and that number of blocks matches number of currencies set-up for deposits
 * Ensure that block design matches one in ITsL0PJ0Xz4ZzQpsWyMgiI-fi-1565%3A1798

-------------------
